app/services/eyeshade/publishers.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Eyeshade::Publishers < Eyeshade::BaseApiClient
+  RESOURCE = "/v2/publishers"
+
+  def create_settlement(body: )
+    return {} if Rails.application.secrets[:api_eyeshade_offline]
+
+    connection = Faraday.new(url: api_base_uri) do |config|
```

Comment:
Thinking about moving this connection up to the Eyeshade::BaseApiClient @yachtcaptain23 Thoughts?

Reply:
IMO you should be able to copy the code style that uses `Eyeshade::BaseApiClient`. You can see a sample `post` in `uphold_services/card_creation_service.rb`